plugins/processors/reversedns/parallel/unordered.go

```diff
@@ -0,0 +1,58 @@
+package parallel
+
+import (
+	"context"
+	"sync"
+
+	"github.com/influxdata/telegraf"
+	"golang.org/x/sync/semaphore"
+)
+
+type Unordered struct {
```

Comment:
Is there a requirement to output metrics in the order they arrived?  I see that there are ordered and unordered worker pool types but only ordered is used.  If processors are required to output in order, we should remove the unordered one.  In that case the Parallel interface isn't needed either.

Reply:
I think the expectation is at the moment that they stay ordered. We built this as a proof of concept to show that we had a reasonable architecture, and with the idea that we may want to expose this ordered/unordered choice to the user. Some users definitely seem to care, and some would trade the ordered constraint for better throughput.

Reply:
When we talked about this a few days ago I understood that we were going to default to unordered but have a config option to allow the user to choose ordered.  Is that still the plan?

Reply:
Ah. I forgot about this. Ok, will do.